tasks/bump.js

```diff
@@ -60,6 +81,11 @@ module.exports = function(grunt) {
     var next = function() {
       if (!queue.length) {
         grunt.config.set('bump.version', globalVersion);
+
+        // reset original cwd
+        if (originalCwd) {
+          grunt.file.setBase(originalCwd);
```

Comment:
wont this mess up the file list for every exec after the first? Should the original changeBase go in the runIf?